packages/tar-parser/README.md

```diff
@@ -44,6 +46,34 @@ await parseTar(response.body, { filenameEncoding: 'latin1' }, (entry) => {
 });
 ```
 
+### Working with Sparse Files
+
+For sparse files, tar-parser reconstructs the file by default, filling in zeroed regions as indicated by the sparse map:
+
+```ts
+await parseTar(response.body, async (entry) => {
+  if (entry.header.type === 'sparse') {
+    // Fully reconstructed sparse file
+    let reconstructedData = await entry.bytes();
+    console.log(entry.name, reconstructedData.length);
+  }
+});
+```
+
+If you prefer the raw data chunks as they appear in the archive (without reconstructing zeros), you can call `entry.bytes({ raw: true })`:
```

Comment:
This is the default behavior of `tar`.

Basically, if someone sends you a sparse tarball, you don't have to do anything differently if you just pipe to a write stream for the file. So, for someone with no knowledge of sparse files, it "just work".

However, if you're a more advanced user, you can get the sparse offset's and lengths, and use `fs.write(fd, buffer, offset, length)`, which will be more efficient, because you're not writing the extra data.